src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -22,36 +26,61 @@
 import org.json.JSONObject;
 import org.openqa.selenium.WebDriverException;
 
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.properties.HeliosConfig;
 
 /**
  * Created by Ludwik on 2015-08-05.
  */
 public class Helios {
 
-  public static HashMap<String, String> tokenCache = new HashMap<String, String>();
-
+  private static final Map<String, String> tokenCache = new HashMap<String, String>();
+  private static final String IOEXCEPTION_ERROR_MESSAGE = "PLEASE CHECK IF YOUR VPN IS ENABLED";
+  private static RequestConfig requestConfig = RequestConfig.custom().setConnectTimeout(3000)
+      .setSocketTimeout(3000).build();
 
-  private Helios() {
-  }
+  private Helios() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)